src/CpmsCommon/Controller/Plugin/SendPayload.php

```diff
@@ -18,29 +20,31 @@
  */
 class SendPayload extends AbstractPlugin
 {
-    const API_VERSION_KEY = 'api_version';
+    public const API_VERSION_KEY = 'api_version';
 
     /**
      * Set HTTP status code if specified and return appropriate view model
      *
-     * @param $payLoad
+     * @param array|object $payLoad
      *
      * @return \Laminas\View\Model\ModelInterface
      */
     public function __invoke($payLoad)
     {
-        /** @var \Laminas\Http\PhpEnvironment\Response $response */
-        /** @var RouteMatch $routeMatch */
-        /** @var Application $application */
         $controller  = $this->getController();
         $serviceLocator = $controller->getServiceLocator();
 
-        $payLoad     = (array)$payLoad;
-        $config      = $serviceLocator->get('config');
+        $payLoad = (array)$payLoad;
+        /** @var array */
```

Comment:
why sometimes we provide the variable name and sometimes not in doc strings?

Reply:
I've gone through and standardised it to a single line with the variable name. Class vars now have the type as part of the declaration, not in a comment.